cmd/nvidia-mig-manager/main.go

```diff
@@ -338,7 +384,7 @@ func runScript(migConfigValue string) error {
 		"-p", defaultGPUClientsNamespaceFlag,
 	}
 	if cdiEnabledFlag {
-		args = append(args, "-e", "-t", driverRoot, "-a", driverRootCtrPath, "-b", devRoot, "-j", devRootCtrPath)
+		args = append(args, "-e", "-t", driverRoot, "-a", driverRootCtrPath, "-b", devRoot, "-j", devRootCtrPath, "-l", driverLibraryPath, "-q", nvidiaSMIPath)
```

Comment:
the new parameters should be arguments to this function.

Reply:
Done.